00 Core/MWSE/mods/tew/AURA/common.lua

```diff
@@ -178,6 +230,11 @@ function this.isRefSheltered(options)
         return true
     end
 
+    local function log(message)
```

Comment:
I've always found it superfluous for my use-case, but maybe it will come in handy for you?

https://mwse.github.io/MWSE/guides/logging/

Reply:
Same. Also, too lazy for logging levels heh. 3-line function and I get my local logging level